<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthMapDlgClass</class>
 <widget class="QWidget" name="DepthMapDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>深度图</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;传感器采集位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;传感器参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>291</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>291</width>
         <height>111</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>水平最大采样角度：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>角度分辨率：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>垂直最大采样角度：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="angularResolution">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="maxAngleWidth">
           <property name="text">
            <string>360.0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="maxAngleHeight">
           <property name="text">
            <string>180.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;深度图参数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>111</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QLineEdit" name="Y">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>  Z:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>  Y:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="X">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>  X:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Z">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QFrame" name="frame_3">
       <property name="minimumSize">
        <size>
         <width>361</width>
         <height>111</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>361</width>
         <height>111</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>341</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>  最小获取距离：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>  近邻点对查询点距离值影响因子：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>  深度图边缘宽度：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="noiseLevel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="minRange">
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="borderSize">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>图像保存：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Path">
       <property name="text">
        <string>RangeImageRGB.png</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Browse">
       <property name="minimumSize">
        <size>
         <width>112</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>生成</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>112</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
